test/migrations/update-merkle-distributor-rewards-parameters-proposal.ts

```diff
@@ -0,0 +1,121 @@
+import BNJS from 'bignumber.js';
+import { BigNumber, BigNumberish } from 'ethers';
+
+import config from '../../src/config';
+import { getDeployConfig } from '../../src/deploy-config';
+import { getDeployerSigner } from '../../src/deploy-config/get-deployer-address';
+import { log } from '../../src/lib/logging';
+import { waitForTx } from '../../src/lib/util';
+import { impersonateAndFundAccount } from '../../src/migrations/helpers/impersonate-account';
+import { updateMerkleDistributorRewardsParametersProposal } from '../../src/migrations/update-merkle-distributor-rewards-parameters-proposal';
+import { DydxGovernor__factory, DydxToken__factory, MerkleDistributorV1__factory } from '../../types';
+import { advanceBlock, increaseTimeAndMine } from '../helpers/evm';
+
+const MOCK_PROPOSAL_IPFS_HASH = (
+  '0x0000000000000000000000000000000000000000000000000000000000000000'
+);
+
+export async function updateMerkleDistributorRewardsParametersNoProposal({
+  merkleDistributorAddress,
+  shortTimelockAddress,
+}: {
+  merkleDistributorAddress: string,
+  shortTimelockAddress: string,
+}) {
+  const deployConfig = getDeployConfig();
+  const mockShortTimelock = await impersonateAndFundAccount(shortTimelockAddress);
```

Comment:
Suggestion: Follow [these variable naming schemes](https://github.com/dydxfoundation/governance-contracts/blob/4a011f7eeed29f709aca3d4977c525c76eea558e/test/stark-proxy/sp2-guardian.spec.ts#L48) for consistency.

```suggestion
  const shortTimelockSigner = await impersonateAndFundAccount(shortTimelockAddress);
```